packages/packages.csv

```diff
@@ -74,7 +74,7 @@ Microsoft.AspNetCore.Mvc.RazorPages,ship,include
 Microsoft.AspNetCore.Mvc.Razor.ViewCompilation,ship,include
 Microsoft.AspNetCore.Mvc.TagHelpers,ship,include
 Microsoft.AspNetCore.Mvc.ViewFeatures,ship,include
-Microsoft.AspNetCore.Mvc.WebApiCompatShim,ship,include
+Microsoft.AspNetCore.Mvc.WebApiCompatShim,ship,exclude
```

Comment:
We don't expect any apps targeting .NET Core to use this package right @rynowak ?